Rebuild-Black-Business/RBB-Website
# Added Legal Page

# Describe your PR

- Terms and Conditions page now replaced by Legal Page.
- Added link to list in footer. 
- Removed TaC link from footer.
- Updated Ally Gate link and removed isExternal

Related to https://trello.com/c/LEzdrqEV/109-make-legal-for-terms-and-conditions-privacy-policy

## Pages/Interfaces that will change
- Terms and Condition page
- Legal Page
- Footer
- Allies Gate

### Screenshots / video of changes

![image](https://user-images.githubusercontent.com/20157895/84222970-80c26680-aa9e-11ea-8f2c-6f804ef0b02a.png)

![image](https://user-images.githubusercontent.com/20157895/84223127-da2a9580-aa9e-11ea-9b4d-841a5f4eecdd.png)


## Steps to test

1. Go to /legal and see legal docs
2. Looks at footer to see link to Legal and no longer see link to Terms and Conditions
3. Go to /allies and click the link to Terms and Conditions to be taken to Legal
2. Go to /terms-and-conditions and see nothing


### Additional notes
The id in the anchor tag doesn't seem to be working consistently.
